### ISSUE TITLE
Missing Linkbox

### ISSUE THREAD

**Issue by Offerel** [gsantner/markor#856]
#### General information
* **App version: 2.2.6** 
* **System: Android 10** 

#### Description
I must admit, I have rarely used this function before. Recently, however, I have been forced to use another browser, where the function to synchronize bookmarks via a policy is disabled. Apart from the fact that I am not at all enthusiastic about using Chrome, I would still like to be able to exchange bookmarks between PC's and Android again. At least on Android the Linkbox comes back to me, unfortunately it seems it doesn't exist anymore. 

Do we have a chance to get this function back?

On the PC I have to look for something else that can also write to the same file. In the ideal case this should be a browser add-on. If someone knows such a thing, I would be really grateful for a corresponding tip.

**Comment by gsantner** [comment]
See discussion on markor website, thanks.

**Comment by Offerel** [comment]
There was a discussion about this? Couldn't find it. Could you please share the direct url if possible please? Would be nice. In the meantime i use todo.txt... and writing a small php file on my website where i can share a url from a browser, which will read url and tiele and add that to the same todo.txt... should be a quick workaround...

**Comment by ChiefMikeK** [comment]
Maybe this?

> [Markor v1.7 - Custom Fonts, LinkBox with Markdown | gsantner](https://gsantner.net/blog/2019/03/11/markor-release-v1.7-custom-fonts-linkbox-markdown.html#linkbox-changed-to-markdown-format)

**Comment by Offerel** [comment]
Ok, got that description... but this isn't part of the actual Markor anymore... i cant use LinkBox.md as target for shared URL's from a browser. I can use ToDo and QuickNote, where ToDo is the better alternative.

I can understand, that Linkbox is maybe a not often used feature, but maybe we can make it optional instead of removing the function completely. 

Anyhow, i want to share my php snippet so far which i use now as a webservice. I know it could be improved but its only a quick and dirty workaround for now:

```
<?php
$imap_server = "your_imap_server.com";

if (!isset($_SERVER['PHP_AUTH_USER'])) {
    header('WWW-Authenticate: Basic realm="Bookmarks"');
    header('HTTP/1.0 401 Unauthorized');
    echo 'Login canceled by user';
    exit;
}

if (imap_open ("{".$imap_server.":993/imap/ssl/readonly}INBOX", $_SERVER['PHP_AUTH_USER'], $_SERVER['PHP_AUTH_PW'])) {
    $filename = "/path/to/userdirs/".$_SERVER['PHP_AUTH_USER']."/Notes/todo.txt";

    if(!empty($_GET["link"])) {
        $url = $_GET["link"];

        if(!empty($_GET["title"])) {
            $title = $_GET["title"];
        }
        else {
            $title = "unknown";
            echo "No Title specified, set to 'unknown'.";
        }
    
        $task = "\r\n".date("Y-m-d")." [".$title."](".$url.")";
        echo "Added ".$title;
        file_put_contents( $filename , $task, FILE_APPEND);
    }
    else {
        //echo 'No Link specified, exiting now.';
        get_bookmarks($filename);
    }
    
}
else {
    echo 'IMAP Login failed, exiting now.';
    exit;
}

function get_bookmarks($filename) {
    if(file_exists($filename)) {
        $bookmarks = file_get_contents($filename);
        header("Cache-Control: no-cache, must-revalidate");
        header("Expires: Sat, 26 Jul 1997 05:00:00 GMT");
        header('Content-Type: text/markdown; charset=UTF-8');
        header('Content-Disposition: inline; filename="bookmarks.md"');
        echo $bookmarks;
    }
    else {
        echo "File $filename doesn't exist yet, exiting now.";
        exit;
    }
}
?>
```
Save it as file with php extension, change the servername and the path to your todo.txt. Since i make a basic authentication against an existing IMAP user on the specified server, you need at least PHP with IMAP extension running on your webserver. There is much room for improvements, i know that. But thats only written underway in the railway on my smartphone for a quick workaround. I can beautify it later if needed or someone wants to use it.

**Comment by gsantner** [comment]
No this, https://github.com/gsantner/gitalk-gsantner-net/issues/7

TL;DR: Use favourite for LinkBox.md file now, or use QuickNote.
